packages/esm-patient-medications-app/src/api/api.ts

```diff
@@ -23,7 +22,7 @@ export function usePatientOrders(patientUuid: string, status: 'ACTIVE' | 'any')
     'commentToFulfiller,drug:(uuid,display,strength,dosageForm:(display,uuid),concept),dose,doseUnits:ref,' +
     'frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,numRefills,dosingInstructions,' +
     'duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)';
-  const ordersUrl = `${restBaseUrl}/order?patient=${patientUuid}&careSetting=${careSettingUuid}&status=${status}&orderType=${drugOrderTypeUUID}&v=${customRepresentation}`;
+  const ordersUrl = `${restBaseUrl}/order?patient=${patientUuid}&careSetting=${careSettingUuid}&orderTypes=${drugOrderTypeUUID}&v=${customRepresentation}&excludeDiscontinueOrders=true&excludeCanceledAndExpired=true`;
```

Comment:
So, `excludeCanceledAndExpired=true` will essentially ensure that there are no past orders in this request at all, so this query can't be used as a basis for generating both active orders and past orders.